sdk/src/main/java/io/dapr/config/Properties.java

```diff
@@ -68,12 +75,22 @@
       DEFAULT_GRPC_PORT);
 
   /**
-   * Determines if Dapr client will use GRPC to talk to Dapr's side car.
+   * Determines if Dapr client will use gRPC or HTTP to talk to Dapr's side car.
+   */
+  public static final Property<DaprApiProtocol> API_PROTOCOL = new GenericProperty<>(
+      "dapr.api.protocol",
+      "DAPR_API_PROTOCOL",
+      DEFAULT_API_PROTOCOL,
+      (s) -> DaprApiProtocol.valueOf(s.toUpperCase()));
+
+  /**
+   * Determines if Dapr client should use gRPC or HTTP for Dapr's service method invocation APIs.
    */
-  public static final Property<Boolean> USE_GRPC = new BooleanProperty(
-      "dapr.grpc.enabled",
-      "DAPR_GRPC_ENABLED",
-      DEFAULT_GRPC_ENABLED);
+  public static final Property<DaprApiProtocol> API_METHODINVOCATION_PROTOCOL = new GenericProperty<>(
+      "dapr.api.methodInvocation.protocol",

```

Comment:
```suggestion
      "dapr.api.method.invocation.protocol",

```

Reply:
This is wrong because method is not a child of api.